content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -22,13 +22,173 @@ Configuring the [aggregation layer](/docs/concepts/extend-kubernetes/api-extensi
 There are a few setup requirements for getting the aggregation layer working in your environment to support mutual TLS auth between the proxy and extension apiservers. Kubernetes and the kube-apiserver have multiple CAs, so make sure that the proxy is signed by the aggregation layer CA and not by something else, like the master CA.
 {{< /note >}}
 
+{{% /capture %}}
+
+{{% capture authflow %}}
```

Comment:
This tag isn't recognized by Hugo, the static site generator we use to build the docs. So the build ignores everything inside this capture tag. (Note that the change to the heading below does show up on the live site.) The tags are part of the template for the different topic types that we require; they can't be used with arbitrary strings to designate a section type.

Reply:
fyi @Bradamant3 just done that on my local dev env and removing

`{{% capture authflow %}}` as well as closing `{{% /capture %}}` didn't make any difference. hmm ..

Reply:
one comment @Bradamant3 is that i run a quick test on my local dev env and i can confirm that by removing `{{% capture authflow %}}` and `{{% /capture %}}` does solve it however the text does show **_after_** the `Enable Kubernetes Apiserver flags` although in the file the new text added by @deitch is **_before_**

Reply:
Good catch, @DanyC97, and thanks very much for testing -- my suggestion to remove the offending capture tags was incomplete. If the added content needs to show up in the order specified in the markdown file, it needs to be placed inside the `steps` capture tag. See https://kubernetes.io/docs/contribute/style/page-templates/#task-template.

Reply:
@deitch should you not have time/ spare cycle to fix this one, let me know and i'll take over

Reply:
Thanks @DanyC97 . I must have copied it over from somewhere when doing the previous PR and it lingered. I will clean it up and update.

>  should you not have time/ spare cycle to fix this one, let me know and i'll take over

Never have, but I will do it anyways. :-)

Reply:
Fixed that and updated. Please check it.

Reply:
I fixed the tags on the open PR that I have at #12890 since it was open anyways. Take a look there?